### PR TITLE
Multiple Queries and Join Operations

### PR DESCRIPTION
**Pull request by peterdtitan**
In this branch, I have updated the database to contain a vets table, a specialization table, and a visits table. With the added input, I performed the following queries on the database: 

- [ ] Who was the last animal seen by William Tatcher?

- [ ] How many different animals did Stephanie Mendez see?

- [ ] List all vets and their specialties, including vets with no specialties.

- [ ] List all animals that visited Stephanie Mendez between April 1st and August 30th, 2020.

- [ ] What animal has the most visits to vets?

- [ ] Who was Maisy Smith's first visit?

- [ ] Details for most recent visit: animal information, vet information, and date of visit.

- [ ] How many visits were with a vet that did not specialize in that animal's species?

- [ ] What specialty should Maisy Smith consider getting? Look for the species she gets the most.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -161,3 +161,97 @@ JOIN owners ON animals.owner_id = owners.id
...
+-- What specialty should Maisy Smith consider getting? Look for the species she gets the most.
+
+SELECT species.name, COUNT(*) AS num_visits
+FROM animals
+JOIN visits ON animals.id = visits.animal_id
+JOIN species ON animals.species_id = species.id
+WHERE animals.owner_id = 2
+GROUP BY species.name
+ORDER BY num_visits DESC
+LIMIT 1;
```
Comment (BregornOriginal) [inline queries.sql]: Good job so far! 🥇 

This is one of the most difficult queries, let´s check what´s wrong

So in this query, we need to find the most visited species name that Maisy Smith gets. You are getting the visits of `Pokemon` but as here we have a small chunk of data, we can easily see that `Digimon` is the most species that this vet receives (6 specifically)

- Kindly refactor the query to get the correct query on this challenge 💪🏼 

You could bring a query like this result as an advice 

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @peterdtitan  

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!

one-step-closer-one-more

## Highlights

✔️ Linters are passing
✔️ README looks professional
✔️ Excellent Pull Request description

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (peterdtitan) [reply queries.sql]: Updated the output to reflect the correct query result: 

Screenshot 2023-05-11 at 4 00 45 PM

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**